bin/ch-validate

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+. $(dirname "$0")/base.sh
+
+usage () {
+   cat 1>&2 <<EOF
+Verify that a specified Charliecloud image has the basic required structure to run.
+
+Usage:
+
+   $ $(basename $0) IMAGE_DIR
+
+Arguments:
+   
+   IMAGE_DIR   Directory containing the Charliecloud image
+
+EOF
+   exit ${1:-1}
+}
+
+need_files=( 
+   "/etc/passwd"
+   "/etc/group"
+   "/etc/hosts"
+   "/etc/resolv.conf"
+)
+
+need_dirs=(
+   "/etc"
+   "/dev"
+   "/proc"
+   "/sys"
+   "/tmp"
+)
+
+
+if [ "$1" = "--help" ]; then
+    usage 0
+fi
+if [ "$1" = "--version" ]; then
+    version
+    exit 0
+fi
+
+if [ $# -ne 1 ]; then
+   usage -1
```

Comment:
I think this can just be `usage`, since it will default to exit code 1, and are negative exit codes a thing? I vaguely recall that the `-` in `${1:-1}` is part of the `:-` operator.